internal/gps/pkgtree/pkgtree.go

```diff
@@ -203,6 +203,12 @@ func fillPackage(p *build.Package) error {
 		if filepath.Base(file)[0] == '_' {
 			continue
 		}
+
```

Comment:
Any file that starts with `.`, `_` or called `testdata` should be ignored when scanning source directories. Rather than this check, extend the check on line 203 to check for `== '.'.